mqt/qecc/cc_decoder/decoder.py

```diff
@@ -11,7 +11,9 @@
 import numpy as np
 from z3 import Bool, Not, Optimize, Xor, simplify
 
+from mqt.qecc.cc_decoder.color_code import ColorCode, LatticeType
 from mqt.qecc.cc_decoder.hexagonal_color_code import HexagonalColorCode
+from mqt.qecc.cc_decoder.square_octagon_color_code import SquareOctagonColorCode
```

Comment:
These classes should all be importable from `cc_decoder`. You can accomplish that by adding the classes to the respective `__init__.py` file.